main/test/ca/uwaterloo/flix/util/FlixTest.scala

```diff
@@ -26,10 +26,7 @@ class FlixTest(name: String, path: String) extends FunSuite {
   /*
    * Include the standard library.
    */
-  val Library: List[String] = List(
```

Comment:
Okay; do you notice any slow down with the unit tests (by including everything)?